src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,62 @@
+import { useContext, useEffect, useState } from 'react';
+import { PeopleContext } from '../store/PeopleContent';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { PeopleFilters } from '../components/PeopleFilters';
+
+export const PeoplePage = () => {
+  const { setPeopleLoader, setPeople, peopleLoader, peopleToShow, people } =
+    useContext(PeopleContext);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  useEffect(() => {
+    setErrorMessage('');
+    setPeopleLoader(true);
+    getPeople()
+      .then(setPeople)
+      .catch(() => {
+        setErrorMessage('Something went wrong');
+      })
```

Comment:
The error handling in the `catch` block sets a generic error message. Consider logging the actual error or providing more specific error messages to help with debugging and user feedback.